src/dmd/root/stringtable.d

```diff
@@ -203,7 +203,7 @@ public:
      * Returns:
      *      last return value of fp call
      */
-    extern (C++) int apply(int function(const(StringValue)*) fp)
+    int apply(int function(const(StringValue)*) fp)
```

Comment:
This needs to be updated as well to have D linkage: https://github.com/dlang/dmd/blob/2d6ca63fba41b09063fad552fd5ffa2370c331cc/src/dmd/dinifile.d#L144.

Reply:
Done.